DataConnectors/CEF/cef_troubleshoot.py

```diff
@@ -291,8 +293,9 @@ def incoming_logs_validations(incoming_port, ok_message, mock_message=False):
     poll_obj = select.poll()
     poll_obj.register(tcp_dump.stdout, select.POLLIN)
     while (end_seconds - start_seconds) < tcpdump_time_restriction:

```

Comment:
Do we still need this while? why not change all the logic to "for 5 times"?

Reply:
Because we have 2 previous tcpdumps (on port 514 and port 25226).
These tcpdumps check if data is flowing in (without any mock generation)
they need those 60 seconds while loop of polling to see if any data is received.